src/parser/MatchPath.h

```diff
@@ -115,6 +115,10 @@ class MatchEdge final {
     return range_.get();
   }
 
+  bool anonymous() const {
+    return alias_.empty() || alias_.front() == '_';
```

Comment:
Better to use `AnonVarGenerator::isAnnoVar()`.
IMHO, we should manage all anonymous variables in query context.